hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/report/model/TenantUsage.java

```diff
@@ -30,81 +35,70 @@ public TenantUsage(final String tenantName) {
         this.tenantName = tenantName;
     }
 
-    /**
-     * @return name of the tenant
-     */
     public String getTenantName() {
         return tenantName;
     }
 
-    /**
-     * @return number of targets of the tenant
-     */
     public long getTargets() {
         return targets;
     }
 
-    /**
-     * @param targets
-     *            of the tenant
-     * @return updated tenant stats element
-     */
     public TenantUsage setTargets(final long targets) {
         this.targets = targets;
         return this;
     }
 
-    /**
-     * @return number of undeleted artifacts of the tenant
-     */
     public long getArtifacts() {
         return artifacts;
     }
 
-    /**
-     * @param artifacts
-     *            of tenant
-     * @return updated tenant stats element
-     */
+    public Map<String, String> getUsageData() {
+        if (usageData == null) {
```

Comment:
Ok. Do you need the `public` getter? 
Normally is better to return a unmodifiable map or make it `private`, because you create a special `addUsageData` method.